src/modules/task-show.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] It is important to note that It is not allowed to disable linter errors because you are just giving room for syntax errors and bugs in the future and that's what linters help us prevent. It is advised to work in other to solve errors flagged by linter checks instead of disabling them 😃 So please remove the `eslint-disable` here and in all your files 😉